internal/contour/listener.go

```diff
@@ -122,6 +127,22 @@ func (lvc *ListenerVisitorConfig) httpsAccessLog() string {
 	return DEFAULT_HTTPS_ACCESS_LOG
 }
 
+// minProtocolVersion returns min protocol version if specified via annotation first
```

Comment:
once set by the operator.. only applicable values should be protocols higher or equal to the one set by the operator
example: lets set operator sets the min to 1.2 users cannot use protocol 1.1
below logic does not seems to achieve this..